wfe2/aia.go

```diff
@@ -0,0 +1,91 @@
+//go:build integration
+
+package wfe2
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+
+	"github.com/honeycombio/beeline-go/wrappers/hnynethttp"
+	"github.com/letsencrypt/boulder/features"
+	"github.com/letsencrypt/boulder/issuance"
+	"github.com/letsencrypt/boulder/metrics/measured_http"
+	"github.com/letsencrypt/boulder/probs"
+	"github.com/letsencrypt/boulder/web"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	aiaIssuerPath = "/aia/issuer/"
+)
+
+// Handler returns an http.Handler that uses various functions for
```

Comment:
Rather than duplicate all of `Handler` between integration and non-integration, I would unconditionally add a `HandleFunc(m, "/aia/issuer/", wfe.Issuer, "GET"` and then define an `Issuer()` that 404s in non-integration, and an `Issuer()` that 200s in integration.

Reply:
Yeah I was vaguely concerned about potentially exposing a new handled URL in prod, but you're right that it's much cleaner. Done.